src/main/scala/xiangshan/mem/lsqueue/LoadQueueData.scala

```diff
@@ -128,14 +140,26 @@ class LqPAddrModule[T <: UInt](
   numWrite: Int,
   numWBank: Int,
   numWDelay: Int = 1,
-  numCamPort: Int = 1)(implicit p: Parameters) extends LqRawDataModule(gen, numEntries, numRead, numWrite, numWBank, numWDelay, numCamPort)
+  numCamPort: Int = 1,
+  enableCacheLineCheck: Boolean = false,
+  paddrOffset: Int = 0 // Used to compare with `DCacheLineOffset`.
```

Comment:
I think it is better not to initialize `paddrOffset` to 0, which maybe misleading. Also, comments are needed.
```suggestion
  paddrOffset: Int // The least significant `paddrOffset` bits of paddr are neglected
```